guvnor-rest/guvnor-rest-backend/src/main/java/org/guvnor/rest/backend/ProjectResource.java

```diff
@@ -134,6 +129,8 @@ public JobResult getJobStatus(@PathParam("jobId") String jobId) {
         return job;
     }
 
+    //TODO: Stop or cancel a job
+
```

Comment:
I wonder when this will get done? Leaving it in there in case someone reads our code and goes "oh right, I'll do that now".